src/status_im/config.cljs

```diff
@@ -34,24 +34,6 @@
 
 (def mainnet-rpc-url (str "https://eth-archival.rpc.grove.city/v1/" POKT_TOKEN))
 (def sepolia-rpc-url (str "https://sepolia-archival.rpc.grove.city/v1/" POKT_TOKEN))
-(def mainnet-chain-explorer-link "https://etherscan.io/address/")
```

Comment:
all the chain explorer urls are coming from status-go already

Reply:
Happy to see this cleanup 🙌